covalent/executor/executor_plugins/dask.py

```diff
@@ -85,6 +94,9 @@ def __init__(
         super().__init__(log_stdout, log_stderr, conda_env, cache_dir, current_env_on_conda_fail)
 
         self.scheduler_address = scheduler_address
+        self.workdir = workdir
+        set_config("executors.dask.workdir", self.workdir)
```

Comment:
We don't wanna change the values set in the config file every time whenever the executor is instantiated as they are more of a "global" value than specific.

Reply:
Thanks, won't be calling `set_config`